index.html

```diff
@@ -0,0 +1,135 @@
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+
+      <section class="preamble">
+        <div class="preamble__inner">
+          <h2 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h2>
+          <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
+        </div>
+      </section>
+
+      <section class="place-one place-one--blue">
+        <div class="place-one__inner">
+          <h2 class="place-one__header">The Present Town</h2>
+          <p class="place-one__counter-paragraph">— №1 —</p>
+          <p class="place-one__paragraph">Sedona is not a tourist attraction, the life floows in here in it own way.</p>
+          <img class="place-one__image" src="" alt="Building in the city"/>
+        </div>
+      </section>
+
+      <section class="recommendations">
+        <div class="recommendations__inner">
+          <ul class="recommendations__list">
+            <li class="recommendations__item">
+              <!--house-pic-->
+              <h2 class="recommendations__header">Accomodation</h2>
+              <p class="recommendations__paragraph">We recommend to live in a real motel, everything is like in a movie!</p>
+            </li>
+            <li class="recommendations__item">
+              <!--gift-pic-->
+              <h2 class="recommendations__header">Souvenirs</h2>
+              <p class="recommendations__paragraph">Not only Chinese, but also the local products!</p>
+            </li>
+            <li class="recommendations__item">
+              <!--burger-pic-->
+              <h2 class="recommendations__header">Food</h2>
+              <p class="recommendations__paragraph">Always order a Brand Burger, you will not be disappointed!</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="place-two place-two--blue">
+        <div class="place-two__inner">
+          <h2 class="place-two__header">There's a Devil's Bridge</h2>
+          <p class="place-two__counter-paragraph">— №2 —</p>
+          <p class="place-two__paragraph">Yes, you can go through it! If you dare of course.</p>
+        </div>
+        <img class="place-two__image" src="" alt="Devil's bridge in Sedona."/>
+      </section>
+
+      <section class="place-three">
```

Comment:
change the class name, and places 3,4,5, I think should be wrapped in a container, as they are forming a set, flex-box will require it, but you can add later when you'll be styling it for tablet